src/sentry/api/event_search.py

```diff
@@ -36,17 +36,29 @@
 
 event_search_grammar = Grammar(
     r"""
-search            = (boolean_operator / paren_term / search_term)*
-boolean_operator  = spaces (or_operator / and_operator) spaces
-paren_term        = spaces open_paren spaces (paren_term / boolean_operator / search_term)+ spaces closed_paren spaces
-search_term       = key_val_term / quoted_raw_search / raw_search
-key_val_term      = spaces (time_filter / rel_time_filter / specific_time_filter
-                    / duration_filter / boolean_filter / numeric_filter
-                    / aggregate_filter / aggregate_date_filter / aggregate_rel_date_filter
-                    / has_filter / is_filter / text_filter)
-                    spaces
-raw_search        = (!key_val_term ~r"\ *(?!(?i)OR(?![^\s]))(?!(?i)AND(?![^\s]))([^\ ^\n ()]+)\ *" )*
-quoted_raw_search = spaces quoted_value spaces
+search = (boolean_operator / paren_term / search_term)*
+
+boolean_operator = spaces (or_operator / and_operator) spaces
+paren_term       = spaces open_paren spaces (paren_term / boolean_operator / search_term)+ spaces closed_paren spaces
+search_term      = key_val_term / free_text_quoted / free_text
+
+free_text        = (!key_val_term ~r"\ *(?!(?i)OR(?![^\s]))(?!(?i)AND(?![^\s]))([^\ ^\n ()]+)\ *" )*
```

Comment:
This is the original raw_search. This doesn't include the actual changes from https://github.com/getsentry/sentry/pull/26141